testutils/build.gradle

```diff
@@ -22,4 +22,8 @@ dependencies {
     implementation dependency.junit
     implementation dependency.androidx.test.core
     implementation dependency.rxjava
+
+    // dependency on Model/GraphQL integration classes
+    // remove when modules are re-organized to provide better isolation
+    compileOnly project(path: ':aws-api')
```

Comment:
Ah! It's over _here_, now. :-D This doesn't bother me as much though. Being a purist about our dependency hierarchy in test code is less important. 

Not for this PR --

Probably, though, we should migrate `SyncrhonousApi` so that it only contains methods that resemble `ApiCategoryBehavior`, sans-callbacks. 

Like:
```
void asyncFoo(Bar bar, Consumer<Ok> onOk, Conusmer<NotOk> onNotOk);
```
-->
```
Ok syncFoo(Bar bar) throws NotOk
```

Reply:
>Probably, though, we should migrate SyncrhonousApi so that it only contains methods that resemble ApiCategoryBehavior, sans-callbacks.

I like that idea! Make it implement the `ApiCategoryBehavior` so it adheres to the contract.